pkg/logstash.sysv

```diff
@@ -114,6 +114,27 @@ force_stop() {
   fi
 }
 
+config_test() {
+  # Check if a config file exists
+  if [ ! "$(ls -A $LS_CONF_DIR/*.conf 2> /dev/null)" ]; then
```

Comment:
I missed that. I was used to other programs requiring .conf in order to load the file (and they were skipping everything that didn't contain that specific extension). I'll change it to wildcard.